lib/pipefy_message/publisher.rb

```diff
@@ -14,8 +14,12 @@ def initialize(broker = "aws", broker_opts = {})
       @publisher_instance = build_publisher_instance
     end
 
-    def publish(message, topic, context = nil)
-      @publisher_instance.publish(message, topic, context)
+    def publish(message, topic, context = nil, correlation_id = nil)
+      context = "NO_CONTEXT_PROVIDED" if context.nil?
+      correlation_id = "NO_CID_PROVIDED" if correlation_id.nil?
+      event_id = SecureRandom.uuid.to_s
```

Comment:
**suggestion: ** How about merging considering both `correlation_id` and `event_id` as attributes of `context` hash? This way we wouldn't need to provide this two extra properties to `PipefyMessage::Loggin#log_context`.